project.clj

```diff
@@ -101,7 +101,7 @@
                        :dependencies [[puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]]}
              :ci {:plugins [[lein-pprint "1.1.1"]]}}
 
-  :test-selectors {:default (complement :integration)
+  :test-selectors {:default (constantly :all)
```

Comment:
Nope.  When I make that change, this is what happens:
```
$ lein test

lein test user

Ran 0 tests containing 0 assertions.
0 failures, 0 errors.
```
Looks like the value in this map needs to be a function, not just a keyword.